plugins/catalog-react/report-alpha.api.md

```diff
@@ -381,6 +423,29 @@ export type EntityPredicateValue =
       $contains: EntityPredicateExpression;
     };
 
+// @alpha (undocumented)
+export type FactoryDialogParams = {
+  useOnClick: () => React_2.MouseEventHandler<HTMLLIElement>;
+  useTitle: () => string;
+  icon: React_2.JSX.Element;
+  useIsDisabled?: () => boolean;
+};
+
+// @alpha (undocumented)
+export type FactoryHrefParams =
+  | {
+      useTitle: () => string;
+      icon: React_2.JSX.Element;
+      useHref: () => string;
+      useIsDisabled?: () => boolean;
+    }
+  | {
+      useTitle: () => string;
+      icon: React_2.JSX.Element;
+      href: string;
+      useIsDisabled?: () => boolean;
+    };
+
```

Comment:
These parameters are no longer needed with useProps, right?

Reply:
Correct, thanks for flagging. I've removed them in the latest commit.